### ISSUE TITLE
FlightRadar24 Entered area triggered at incorrect location

### ISSUE THREAD

**Issue by SingingDwarf** [AlexandrErohin/home-assistant-flightradar24#83]
I have installed and configured HA-FR24 with the following settings:

```
Radius in meters
1000
Latitude
52.xx
Longitude
1.yy
Scan interval for updates in seconds
10
The minimum altitude in feet above which all aircraft will be tracked.
-1
The maximum altitude in feet below which all aircraft will be tracked.
5000

```

where xx and yy are two digits for my location.

The 'FlightRadar24 Entered area' trigger is raised for flights entering a location near Toronto, rather than the location configured within the UK.

Furthermore when running the notification shown in the [example](https://github.com/AlexandrErohin/home-assistant-flightradar24?tab=readme-ov-file#notification---when-a-flight-enters-or-exits-your-area), I get the following error:

```
Template variable error: 'dict object' has no attribute 'event' when rendering 'Flight entry of {{ trigger.event.data.callsign }} to {{ trigger.event.data.airport_destination_city }} [Open FlightRadar](https://www.flightradar24.com/{{ trigger.event.data.callsign }})'
Template variable error: 'trigger' is undefined when rendering 'Flight entry of {{ trigger.event.data.callsign }} to {{ trigger.event.data.airport_destination_city }} [Open [Open FlightRadar](https://www.flightradar24.com/{{ trigger.event.data.callsign }})'
Template variable error: 'dict object' has no attribute 'event' when rendering 'Flight entry of {{ trigger.event.data.callsign }} to {{ trigger.event.data.airport_destination_city }} [Open [Open FlightRadar](https://www.flightradar24.com/{{ trigger.event.data.callsign }})'
Template variable error: 'dict object' has no attribute 'event' when rendering 'Flight entry of {{ trigger.event.data.callsign }} to {{ trigger.event.data.airport_destination_city }} [Open FlightRadar](https://www.flightradar24.com/{{ trigger.event.data.callsign }})'
```

**Comment by SingingDwarf** [comment]
Created wrong automation type

**Comment by AlexandrErohin** [comment]
1) there should be 5 digits after the dot - not 2
2) Are you sure you use right trigger? The error says there is no event in trigger

**Comment by SingingDwarf** [comment]
Looks like this is related to the top 10 tracked flights - disabling this and I get notifications for flights in the correct area

**Comment by AlexandrErohin** [comment]
Could you post your automation code?

**Comment by atlflyer** [comment]
I had a similar problem. I live in the United States, and when I configured the component with my lat/lon it instead was reporting flights that were over a particular spot in the Mediterranean Sea where the latitude and longitude where both equal to my latitude. Disabling the top tracked flights resolved the issue.

**Comment by AlexandrErohin** [comment]
@atlflyer could you post the reporting automation code?

**Comment by AlexandrErohin** [comment]
It has been fixed in v1.23.0
@SingingDwarf @atlflyer Would be great if you could give a star to the repo :) Thanks

**Comment by SingingDwarf** [comment]
Awesome, thank you. Sorry, I never got around to sending the automation code.
Glad you resolved it though.
